rpc_util.go

```diff
@@ -181,6 +183,24 @@ func FailFast(failFast bool) CallOption {
 	})
 }
 
+// CallCodec returns a CallOption that uses the given Codec for the RPC call.
+// This will override the Codec assocated with the ClientConn.
+func CallCodec(codec Codec) CallOption {
+	return beforeCall(func(c *callInfo) error {
+		c.codec = codec
+		return nil
+	})
+}
+
+// CallContentSubtype returns a CallOption that will set the content subtype
+// to the given string. Normally, the value returned from Codec.String() is used.
+func CallContentSubtype(contentSubtype string) CallOption {
```

Comment:
Um. Wouldn't it be better to just extend the `Codec` interface to have a method that exposes the `Subtype`? Otehrwise using this will be kinda hard.